flexmark-coverage/pom.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.acrolinx.flexmark</groupId>
+    <artifactId>flexmark-java</artifactId>
+    <version>0.70.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>flexmark-coverage</artifactId>
+  <name>flexmark-java coverage</name>
+  <description>An aggregator of all Jacoco test reports</description>
+
+  <dependencies>
+    <dependency>
```

Comment:
Do we really need to list all these dependencies here, although most depend on each other? Could we not omit transitive ones?

Reply:
Unfortunately, this isn't possible since each has an individual coverage and the purpose is to aggregate all of them, so the dependencies don't actually matter.